src/browser-connection/gateway.js

```diff
@@ -64,6 +66,7 @@ export default class BrowserConnectionGateway {
     static onHeartbeat (req, res, connection) {
         if (BrowserConnectionGateway.ensureConnectionReady(res, connection)) {
```

Comment:
Then I think we should call `prevent` in the `_dispatch` function [here](https://github.com/AlexanderMoskovkin/testcafe-phoenix/blob/9bdd1f2c29f0139808d26ea5297a603297bb9ebc/src/browser-connection/gateway.js#L21) to prevent caching whether the `connection` exists or not